DOCUMENTATION.md

```diff
@@ -68,20 +69,51 @@ Also when running the pipeline Makefile will create additional files:
 
 ## Geocint open-source installation and first run guide
 
-### Installation
+### How to create your custom part of pipeline
 Before the installation of your own geocint pipeline instance, you should create a repository to store your own part of the pipeline.
-Your repository should contain the following required files:
-- install.sh (use [runner-install.sh](runner-install.sh) as an example, store installation of your additional dependencies)
-- Makefile (use [private_make.sample](your_make.sample) as an example; file to store your own additional targets chains. 
 
-Your Makefile should start with export block:
+Since geocint consists of 3 main parts (geocint-runner, geocint-openstreetmap and your custom part) to launch the pipeline
```

Comment:
why 3 parts? I think is should rather be N repositories or geocint-runner and geocint-openstreetmap should be merged together. if geocint-runner and geocint-openstreetmap are tightly connected between each other and every set up requires to have both of them what is the point of having them in separate repositories.